### PR TITLE
clean up typos, comments, TODO markers ...

### PR DESCRIPTION
**Pull request by nickboldt**
clean up typos, comments, TODO markers, spacing errors, engrish, etc.

### CODE REVIEW

**Comment by maxandersen** [comment]
this does not rebase cleanly with master.

can you get it rebased please ?

**Comment by nickboldt** [comment]
Rebased to master.

File: bzira.py [hunk 1]
```diff
@@ -370,14 +355,13 @@ def proces(bug, bugs):
     
 bz = bugzilla.Bugzilla(url=bzserver + "bugs/xmlrpc.cgi")
 
-print "[DEBUG] " + "Querying bugzilla: " + query
+# use chfieldfrom instead of last_change_time so that the URL shown with buglist.cgi returns the same results as the xmlrpc.cgi query
+print "[DEBUG] " + "Querying bugzilla: " + query.replace("last_change_time","chfieldfrom")
```
Comment (nickboldt) [inline bzira.py]: You may recall this fix from #17 when I tried to convince you that a valid, clickable console URL was a good idea. Well, it's back. And it's better than ever.

File: bzira.py [hunk 2]
```diff
@@ -122,11 +119,11 @@ def map_linuxtools(version):
 bzprod_version_map = {
     "WTP Incubator" : (lambda version: NO_VERSION),
 
-    # TODO ensure this works for 3.8.x -> Neon.x 
+    # TODO: ensure this works for 3.8.x -> Neon.x
```
Comment (maxandersen) [inline bzira.py]: much easier to *not* add : since that is actually what we use elsewhere

File: bzira.py [hunk 3]
```diff
@@ -173,21 +166,19 @@ def bz_to_jira_version(options, bug):
...
 
-
 bz2jira_status = {
            "NEW" : "Open",
            "REOPENED": "Reopened",
            "RESOLVED" : "Resolved",
            "VERIFIED" : "Verified",
            "CLOSED" : "Closed",
-           "ASSIGNED" : "Coding In Progress", # is this the right approximation ?
+           "ASSIGNED" : "Coding In Progress", # TODO: determine if this is a valid approximation
```
Comment (maxandersen) [inline bzira.py]: this is a valid approximation, but it is not necessarily the right one. Please don't change the meaning to something different.

File: bzira.py (hunk 1 above)
Comment (maxandersen) [inline bzira.py]: and its still not happening this way.

I've made it right in latest commit.

**Comment by maxandersen** [comment]
hand merged with adjustments so it does not change the meaning of the comments/code.

**Pull request closed without merging**